README.md

```diff
@@ -1,35 +1,30 @@
-# Database
-Discord bot database
+# pgadmin4 and Postgres server
 
-## Initialization
-To start the datebase, type command:
+To launch docker containers with Postgres and pgadmin4 run command:
+```bash
+docker-compose up -d
 ```
-$ docker-compose up -d
+Then open your browser and go to the url:
 ```
-If there is no postgresql container, docker will automatically create new
-container named `postgresovisco`, otherwise local container will run.
-
-At start, change default password. Go into container's shell by command:
+http://localhost:5555
 ```
-$ docker exec -it postgresovisco bash
+E-mail:
 ```
-and enter the PSQL client by typing (`admin` is the main database user):
+esovisco@pg.edu.pl 
 ```
-psql -U admin
+Password
 ```
-From here, enter `\password` to change the current password for user admin. Of
-course, share the new password with your coworkers ;) . At the end, type `exit`
-(works both for leaving psql and shell).
-
-## Finding out the address of the server
+1qazaq1
+ ```
+ After succesfull logging click Servers-> Register -> Server
 
-Use this command to find out what the **local** IP address of the Postgres database
-is.
+ ![image](https://cdn.discordapp.com/attachments/1044342494143451166/1053010730733027468/image.png)
 
-```
-docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' postgresovisco
-```
+Give the name of the  database.
+To connect client with database you have to know the ip of database.
+You can find it by typing `docker inspect <id of docker container>` in the 
```

Comment:
To było wcześniej w readme, myślę że to lepsza metoda na znalezienie adresu IP bazy
```suggestion
You can find it by typing `docker inspect  -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' postgresovisco`
```

Reply:
@kapselccc